src/activities/associateGrade/AssociateGradeEntity.js

```diff
@@ -176,4 +195,14 @@ export class AssociateGradeEntity extends Entity {
 		if (!returnedEntity) return;
 		return new AssociateGradeEntity(returnedEntity);
 	}
+
+	_performActionWithWorkingCopy(action) {
```

Comment:
might want to make this clear that it is only a helper function for GET actions.
PATCH/POST/DELETE works fine.